docs/changelog.md

```diff
@@ -1,6 +1,12 @@
 (changelog)=
 # Release notes
 
+## Verison 0.17.0
+*Unreleased*
+
+- Removed iodide-specific code in `pyodide.js`. This breaks compatibility with
+  iodide and other users pretending to be iodide.
```

Comment:
I think

```suggestion
  iodide.
```
would be fine. Could we maybe add a badge "Breaking change" in the beginning as in https://scikit-learn.org/stable/whats_new/v0.24.html ?